README.md

```diff
@@ -1,3 +1,15 @@
+<!--
```

Comment:
I don't think it is necessary to have this header, it would very likely not be updated

Reply:
Hmm...You're right, there's a feature in git to check who had modified the file, I'll delete it.

Reply:
Fixed.